src/cogent3/parse/blast_xml.py

```diff
@@ -181,29 +177,68 @@ class BlastXMLResult(BlastResult):
     HIT_LENGTH = "HIT_LENGTH"
     SCORE = "SCORE"
     POSITIVE = "POSITIVE"
+    ITERATION = "ITERATION"
+    QUERY_ID = "QUERY ID"
+    SUBJECT_ID = "SUBJECT_ID"
+    PERCENT_IDENTITY = "% IDENTITY"
+    ALIGNMENT_LENGTH = "ALIGNMENT LENGTH"
+    MISMATCHES = "MISMATCHES"
+    GAP_OPENINGS = "GAP OPENINGS"
+    QUERY_START = "Q. START"
+    QUERY_END = "Q. END"
+    SUBJECT_START = "S. START"
+    SUBJECT_END = "S. END"
+    E_VALUE = "E-VALUE"
+    BIT_SCORE = "BIT_SCORE"
 
     # FieldComparisonOperators = (
     #    BlastResult.FieldComparisonOperators = {
     #        HIT_DEF:(_gt, float)
     #        }
     # .. to be done
 
-    # .. extend HitKeys
-    HitKeys = BlastResult.HitKeys.union(
-        set(
-            [
-                HIT_DEF,
-                HIT_ACCESSION,
-                HIT_LENGTH,
-                SCORE,
-                POSITIVE,
-                QUERY_ALIGN,
-                SUBJECT_ALIGN,
-                MIDLINE_ALIGN,
-            ]
-        )
+    HitKeys = set(
+        [
+            HIT_DEF,
+            HIT_ACCESSION,
+            HIT_LENGTH,
+            SCORE,
+            POSITIVE,
+            QUERY_ALIGN,
+            SUBJECT_ALIGN,
+            MIDLINE_ALIGN,
+            ITERATION,
+            QUERY_ID,
+            SUBJECT_ID,
+            PERCENT_IDENTITY,
+            ALIGNMENT_LENGTH,
+            MISMATCHES,
+            GAP_OPENINGS,
+            QUERY_START,
+            QUERY_END,
+            SUBJECT_START,
+            SUBJECT_END,
+            E_VALUE,
+            BIT_SCORE,
+        ]
     )
 
+    # standard comparison for each field, e.g.
+    # want long matches, small e-values
+    _lt = lambda x, y: x < y
```

Comment:
delete all these and do a top level import as
```python
from operator import lt as _lt, ...
```
[see standard library docs](https://docs.python.org/3/library/operator.html).